### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by antonilol**
Added missing installation steps. https://github.com/matthewsot/docs-vim/issues/2

### CODE REVIEW

File: README.md
Comment (matthewsot) [inline README.md]: Can we rename this to "Installation (Chrome)"? Just in case we want to add Firefox instructions sometime :)

File: README.md
Comment (matthewsot) [inline README.md:13]: I'm not super familiar with githubusercontent --- should these image files be inside this repository?

**Review by matthewsot — COMMENTED** [review]
Two small nits, but otherwise looks great! Thanks for sending the PR. Once those two are addressed I'm happy to merge.

File: README.md
Comment (antonilol) [reply README.md:13]: I pasted the images in the README file and it automatically but the githubusercontent thing there. You can out them in the repository but it's not needed.

File: README.md
Comment (antonilol) [reply README.md]: I edited it.

**Review by matthewsot — APPROVED** [review]

**Comment by matthewsot** [comment]
Awesome, thanks! Not sure if you can merge or not once I approve, assuming not then I'll merge in ~10m. Thanks again!

**Pull request merged**
